app/connectors/MicroserviceCacheConnector.scala

```diff
@@ -17,21 +17,15 @@
 package connectors
 
 import com.google.inject.Inject
-import config.FeatureSwitchManagementService
 import identifiers.TypedIdentifier
 import play.api.libs.json._
 import play.api.mvc.Result
-import play.api.mvc.Results.Ok
-import play.mvc.Http.Status
 import uk.gov.hmrc.http._
-import utils.Toggles.IsPsaDataShiftEnabled
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class MicroserviceCacheConnector @Inject()(
```

Comment:
This file is not needed anymore. Please use PensionAdminCacheConnector instead. Also, PensionsSchemeCacheConnector is not redundant.

Reply:
fixed